FAQ.md

```diff
@@ -0,0 +1,40 @@
+# FAQ
+
+## Can `xdotool` work with hexadecimal window IDs?
+
+Yes. You can easily check this, by using `wmctrl` as
```

Comment:
Maybe preface this with "Yes, with hexadecimal values that start with 0x, such as 0x0123456AB" ?